src/runmode-netmap.c

```diff
@@ -127,13 +124,15 @@ static int ParseNetmapSettings(NetmapIfaceSettings *ns, const char *iface,
         }
     }
 
+#ifdef USE_NEW_NETMAP_API
     /* we will need the base interface name for later */
     char base_name[IFNAMSIZ];
     strlcpy(base_name, ns->iface, sizeof(base_name));
     if (strlen(base_name) > 0 &&
-            (base_name[strlen(base_name) - 1] == '^' || base_name[strlen(base_name) - 1] == '*')) {
```

Comment:
This change results in a syntax error during compilation because removing line 134 deletes the opening brace "{" for the if() statement. Furthermore, I don't believe lines 134 and 135 should be deleted.

Reply:
Will fix.